### PR TITLE
changed .right class media query to align center

### PR DESCRIPTION
**Pull request by Zivgl66**
hey, 
found out that some headlines containing the ".right" class don't align to the center when media query is active. changed the CSS to match the requested design.

### CODE REVIEW

**Comment by karliss** [comment]
Can you show a before after screenshot for what you are getting?

I did a quick test and for me it looked like it makes things worse.

**Comment by Zivgl66** [comment]
Hi,
Sure, as you can see in the screenshot the headline and content is align to
the right and not to the center as it should be.

**Comment by karliss** [comment]
What screenshot? I don't see any screenshot. Looks like you responded using email, maybe github doesn't support attaching images while responding using email and it got lost.

**Comment by ITAYC0HEN** [comment]
Except on the full page (computer monitor), I don't see the differences. And on the widescreen, it introduces a regression that looks worse.

Expected behavior:

On widescreen, it should look like this:
_Text should be aligned to the side of the picture. If the pic is on the left side, then the text should be to its right, but aligned to the left._
image
 
On Mobile, I think it's better to have everything centered.

Like this (modified using Inspect Element):
image


Also, please make sure to check whether all changes you commit are intentional. The Files Changed tab can assist you with that.

**Comment by Zivgl66** [comment]
ok, as explained by ITAYC0HEN , I've adjusted to the necessary changes. please review the code again.

File: assets/css/custom/rizin.css [hunk 1]
```diff
@@ -287,8 +287,8 @@
   margin: auto;
 }
 
-.preview-section__feature .preview-section__feature__description.right  {
-  text-align: right;
+.preview-section__feature .preview-section__feature__description.right {
+  text-align: center;
```
Comment (karliss) [inline assets/css/custom/rizin.css:291]: This rule is redundant. Read 5 lines up.

**Comment by ITAYC0HEN** [comment]
Thanks for the contribution! The only change you need to do from the current production situation is to take the existing

```
.preview-section__feature .preview-section__feature__description.right  {
  text-align: right;
}
```

and put it inside the media query.

Give it a try :)

**Comment by Zivgl66** [comment]
fixed review changes.

**Review by ITAYC0HEN — APPROVED** [review]
Thank you Ziv for spotting the issue and taking the time to fix it! Very nice!

**Pull request merged**